### ISSUE TITLE
crontab: = character in the command line will be parsed as a variable

### ISSUE THREAD

**Issue by vrince** [kellyjonbrazil/jc#387]
Let's assume you have a crontab line like : 

```txt
* * * * * /bin/bash -l  -c "python3 my_script.py --output-file=/opt/output.json"
```

Currently `jc` returns:

```json
{
  "variables": [
    {
      "name": "* * * * * /bin/bash -l  -c \"python3 my_script.py --output-file",
      "value": "/opt/output.json\""
    }
  ],
  "schedule": []
}
```

and we expect : 

```json
{
  "variables": [],
  "schedule": [
    {
      "minute": [
        "*"
      ],
      "hour": [
        "*"
      ],
      "day_of_month": [
        "*"
      ],
      "month": [
        "*"
      ],
      "day_of_week": [
        "*"
      ],
      "command": "/bin/bash -l -c \"python3 my_script.py --output-file=/opt/output.json\""
    }
  ]
}
```

To reproduce : 

```sh
echo "* * * * * /bin/bash -l -c \"python3 my_script.py --output-file=/opt/output.json\"" | jc --crontab -p
```

**Comment by kellyjonbrazil** [comment]
Thanks for reporting this - I'll take a look.

**Comment by kellyjonbrazil** [comment]
Hi - I think I have this fixed in `dev`:
https://github.com/kellyjonbrazil/jc/blob/dev/jc/parsers/crontab.py
https://github.com/kellyjonbrazil/jc/blob/dev/jc/parsers/crontab_u.py

You can test by copying the file to your plugin folder:

- Linux/unix: `$HOME/.local/share/jc/jcparsers`
- macOS: `$HOME/Library/Application Support/jc/jcparsers`
- Windows: `$LOCALAPPDATA\jc\jc\jcparsers`

Let me know if you run into any issues! thanks

**Comment by kellyjonbrazil** [comment]
Released in v1.23.2
